### PR TITLE
Update: `H2IK.JoystickGremlin` v13.1.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21194)

### CODE REVIEW

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: H2IK.JoystickGremlin
+PackageVersion: 13.3.0
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: H2IK.JoystickGremlin
+PackageVersion: 13.3.0
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml:5]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: H2IK.JoystickGremlin
+PackageVersion: 13.3.0
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.yaml:5]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+License: GNU General Public License v3.0
+LicenseUrl: https://raw.githubusercontent.com/WhiteMagic/JoystickGremlin/develop/LICENSE
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: A tool for configuring and managing joystick devices.
+Description: Joystick Gremlin is a program that allows the configuration of joystick like devices, similar to what CH Control Manager and Thrustmaster's T.A.R.G.E.T. do for their respectively supported joysticks. However, Joystick Gremlin works with any device be it from different manufacturers or custom devices that appear as a joystick to Windows. Joystick Gremlin uses the virtual joysticks provided by vJoy to map physical to virtual inputs and apply various other transformations such as response curves to analogue axes. In addition to customizing joysticks, Joystick Gremlin also provides powerful macro functionalities, a flexible mode system, scripting using Python, and many other features.
+Moniker: joystick-gremlin
+Tags:
+- joystick
+- virtual inputs
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.locale.en-US.yaml:23]:
```suggestion
- virtual-inputs
```

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+MinimumOSVersion: 10.0.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/WhiteMagic/JoystickGremlin/releases/download/Release_13.3/Joystick.Gremlin.R13.3.msi
+  InstallerSha256: B1A2CC9842DF3A876F3481234D0593CB764341F609A4202F7F70DF71A348759D
+#  ProductCode:
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml:14]:
```suggestion
  ProductCode: "{0DAD4221-C8CF-4424-8DCD-3886274E89EF}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml (hunk 5 above)
Comment (vedantmgoyal2009) [reply manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml]: How to find ProductCode?

Comment (ItzLevvie) [reply manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml]: In the registry, `\Microsoft\Windows\CurrentVersion\Uninstall`.

Comment (vedantmgoyal2009) [reply manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml]: Ok

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/21194#pullrequestreview-708415809

**Comment by vedantmgoyal2009** [comment]
> [#21194 (review)](https://github.com/microsoft/winget-pkgs/pull/21194#pullrequestreview-708415809)

It is version 13.3.0. See here: https://github.com/WhiteMagic/JoystickGremlin/releases/tag/Release_13.3.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
It still has to match Control Panel: https://github.com/microsoft/winget-pkgs/pull/21194#discussion_r671272675.

You'll have to contact the application's developer if the version doesn't seem right.

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: H2IK.JoystickGremlin
+PackageVersion: 13.3.0
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml:5]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.yaml:5]:
```suggestion
PackageVersion: 13.1.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: H2IK.JoystickGremlin
+PackageVersion: 13.3.0
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.yaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.yaml:5]:
```suggestion
PackageVersion: 13.1.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**